### PR TITLE
improve notification toast when saving a file #9803

### PR DESCRIPTION
**Pull request by rdsingh13**
Closes https://github.com/JabRef/jabref/issues/9803

I commented Saving library because its take no time in saving library so only one notification came Library saved 
### Mandatory checks

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by calixtus** [comment]
Hi, thanks for your interest in jabref and open source programming.
Is there any issue linked to this pr you are fixing? Could you elaborate a bit on this description, whats the reasoning for your change?

**Comment by calixtus** [comment]
Just saw the PR title.
Please note that GitHub does not parse pr titles, only the description of your pr. So if you automatically want to link your pr to an issue, you have to write fixes or closes followed by the issue number in the description. Like this:
Closes #9803

**Comment by calixtus** [comment]
After reading through your changes and the issue description:
If you want to remove code, remove it, not just comment it. To keep a history of code changes is what git is for.
Second, the issue comments indicated, that there has not been yet a decision made on how to deal with the notifications. I will put a dev call tag on this one therefor.

**Comment by Siedlerchr** [comment]
Do not remove this. for lage libraries it will take some time

File: src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java [hunk 1]
```diff
@@ -217,7 +217,7 @@ private boolean save(BibDatabaseContext bibDatabaseContext, SaveDatabaseMode mod
 
     private boolean save(Path targetPath, SaveDatabaseMode mode) {
         if (mode == SaveDatabaseMode.NORMAL) {
-            dialogService.notify("%s...".formatted(Localization.lang("Saving library")));
```
Comment (koppor) [inline src/main/java/org/jabref/gui/exporter/SaveDatabaseAction.java:220]: This should be guarded by an if. Constant SHOW_SAVING_SIZE_LIMIT 2_000. The size should be availble from the database context.

**Review by koppor — CHANGES_REQUESTED** [review]

**Review by Siedlerchr — APPROVED** [review]

**Review by Siedlerchr — APPROVED** [review]

**Review by koppor — APPROVED** [review]

**Pull request merged**